gradle-plugin/doc/src/main/asciidoc/inc/scenarios/_quarkus.adoc

```diff
@@ -0,0 +1,74 @@
+[[quarkus-scenario]]
+== Quarkus
+JKube provides ready to use generators. A JKube generator is composed of default configurations to build container images, generate k8s resources and deploy them to kubernetes clusters. There are predifined generators for each type of Java project. For instance, we have a generator for Quarkus projects. It helps JKube to identify that a project is a Quarkus one and provides the right configuration to be used.
+
+This is an example of how you can use the JKube zero configuration for a Quarkus project.
+
+[[quarkus-scenario-prerequisites]]
+=== Prerequisites
+You will need the following for the scenario:
+
+* `minikube` installed and running on your computer
+* `minikube` ingress addon enabled
+
+    $ minikube addons enable ingress
+
+* Use the docker daemon installed in minikube
+
+    $ eval $(minikube -p minikube docker-env)
+
+[[quarkus-scenario-zero-configuration]]
+=== Zero configuration
+1. Start by generating a new Gradle Quarkus project from https://code.quarkus.io. Make sure Gradle is selected. Download the generated project and extract the content.
+
+2. Open the `build.gradle` file and add the plugin in the `plugins` section.
+
+   plugins {
+     id 'java'
+     id 'io.quarkus'
+     id 'org.eclipse.jkube.kubernetes' version '1.6.0-SNAPSHOT'
```

Comment:
```suggestion
     id 'org.eclipse.jkube.{pluginExtension}' version '{version}'
```

Reply:
![Selection_579](https://user-images.githubusercontent.com/650571/146418519-a2c0f09b-8667-46bb-bfb6-39d48ab4b2b4.png)
it doens't seem to work

Reply:
ok found how to fix it

Reply:
fixed